### DISCUSSION TITLE
Entity button.galaxy_tab_a_restart_browser

### DISCUSSION THREAD

**Discussion by splattner** [matt8707/hass-config#77] [Q&A] [answered]
Hey, first of all, awesome project! I'm currently adapting to my house and devices. It looks really great.

One question, where is `button.galaxy_tab_a_restart_browser` generated? Witch entity is creating this? You are using it in the tablet footer https://github.com/matt8707/hass-config/blob/master/popup/footer_tablet.yaml#L132 
Does this come from the browsermod? I can't find any similar entity in my installation nor have I found something in your code. The same for `button.galaxy_tab_a_load_start_url` or `switch.galaxy_tab_a_screensaver`. What am I missing?

**Comment by matt8707** [comment, marked as answer]
Hi, https://github.com/cgarwood/homeassistant-fullykiosk

**Reply by splattner** [reply]
awesome, this I missed! thanks!
